nautobot/extras/validation_engine/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from django.shortcuts import reverse
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+
+from nautobot.core.models import BaseModel
+
+class ValidationResult(BaseModel):
+    class_name = models.CharField(max_length=100)
```

Comment:
I think class_name, function|method_name, last_validation_date, and valid should be required. 

We should also have an `expected_attribute_value` that is not required. This would be in situations where another system is indicating it should be such a value. 


validated_object_attribute => validated_attribute
validated_object_value => validated_attribute_value
+ expected_attribute_value